docs/release_notes/ibexa_dxp_v5.0.md

```diff
@@ -0,0 +1,335 @@
+
+#### Collaboration
+
+With Collaboration, multiple users can invite each other to work on the same content.
+It is a starting point for future functionalities in the collaboration domain.
+
+![Collaboration invite](img/5.0_collaborative_invitation.jpg "Collaboration invite")
+
+For more information, see [Collaboration PHP API](../api/php_api/php_api_reference/php_api_reference/namespaces/ibexa-contracts-collaboration.html) and [Share PHP API](../api/php_api/php_api_reference/php_api_reference/namespaces/ibexa-contracts-share.html).
+
+### Software architecture upgrades
+
+With improved compatibility, performance and increased security, as well as better developer experience in mind, [[= product_name_base =]] decided to introduce several significant tech stack upgrades.
+
+For a full list of updated system requirements, see [Requirements](../getting_started/requirements.md).
+
+#### Symfony 7.3
+
+With this release, [[= product_name =]] moves to Symfony 7.2 from the previously used versions of Symfony.
+
+For details, see [Symfony 7.2](https://symfony.com/doc/current/setup/upgrade_major.html).
+
+#### Doctrine 3.9
+
+By moving to Doctrine 3.9, [[= product_name =]] brings developers better performance, cleaner code, and stronger foundation for a more modern and maintainable application.
+
+#### PHP 8.3
+
+With performance, coding safety and security in mind, with this version, [[= product_name =]] moves to [PHP 8.3](https://www.php.net/releases/8.3/en.php) and drops support for lower versions of the language.
+
+#### OpenAPI support
+
+Adding support for generating the OpenAPI specification for our REST API makes future changes more manageable, and helps our partners automatically generate REST API clients.
+
+Support for serialization and deserialization of REST payloads with the Symfony Serializer component improves data reliability and simplifies debugging.
+
+### Developer experience
+
+#### New packages 
+
+The following packages have been introduced in [[= product_name =]] v5.0.0:
+
+- ibexa/collaboration
+- ibexa/connector-ai
+- ibexa/discounts
+- ibexa/discounts-codes
+- ibexa/product-catalog-date-time-attribute
+- ibexa/product-catalog-symbol-attribute
+- ibexa/share
+
+#### REST APIs
+
+[[= product_name =]] v5.0.0 adds REST API coverage for the following features:
+
+- AI Actions:
+    - [Action Configurations](../api/rest_api/rest_api_reference/rest_api_reference.html#ai-actions-list-action-configurations)
+    - [Action Types](../api/rest_api/rest_api_reference/rest_api_reference.html#ai-actions-list-action-types)
+- [Discounts](/api/rest_api/rest_api_reference/rest_api_reference.html#discounts)
+- [Collaboration](/api/rest_api/rest_api_reference/rest_api_reference.html#collaboration)
+
+#### PHP API
+
+The PHP API has been expanded with the following classes and interfaces:
+
+- [`Ibexa\Contracts\ConnectorAi\Action\Action`](../api/php_api/php_api_reference/classes/Ibexa-Contracts-ConnectorAi-Action-Action.html)
```

Comment:
Preferential, just sharing because you might find it useful: absolute links should work, you can do it like this without worrying about the directory structure.
```suggestion
- [`Ibexa\Contracts\ConnectorAi\Action\Action`](/api/php_api/php_api_reference/classes/Ibexa-Contracts-ConnectorAi-Action-Action.html)
```